src/routes/tasks.ts

```diff
@@ -1,84 +1,69 @@
 import { Router, Request, Response } from 'express';
-import { body, validationResult } from 'express-validator';
-import { Task } from '../models/task';
+import TaskModel from '../models/task';
```

Comment:
If you want to avoid such paths, you can create aliases with `@` in TS where @ could mean your src directory, or you can assign multiple aliases to different folders (i.e. `@routes`, `@models`)

Reply:
https://dev.to/larswaechter/path-aliases-with-typescript-in-nodejs-4353